scripts/dispatch

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
if you switch to alpine or a smaller image you might want to do sh instead of bash to be extra safe

Reply:
These scripts are copy-pasta from another project, rke2-upgrade I think. Bash is already assumed and know to work.